sirepo/job_supervisor.py

```diff
@@ -860,7 +860,7 @@ def res(**kwargs):
                 r.update(self.db.parallelStatus)
                 r.computeJobHash = self.db.computeJobHash
                 r.computeJobSerial = self.db.computeJobSerial
-                r.elapsedTime = self.db.lastUpdateTime - self.db.computeJobStart
+                r.elapsedTime = int(sirepo.srtime.utc_now_as_float()) - self.db.computeJobStart
```

Comment:
I don't think this is right. If the simulation is canceled, the time should not change.